lib/new_relic/agent/instrumentation/elasticsearch.rb

```diff
@@ -0,0 +1,31 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'elasticsearch/instrumentation'
+require_relative 'elasticsearch/chain'
+require_relative 'elasticsearch/prepend'
+
+DependencyDetection.defer do
+  named :elasticsearch
+
+  depends_on do
+    defined?(::Elasticsearch)
+  end
+
+  executes do
+    ::NewRelic::Agent.logger.info('Installing elasticsearch instrumentation')
```

Comment:
The human facing text elsewhere seems to use "Elasticsearch" with a capital "E". Should we use the capitalized "E" here as well, or is it deliberately lowercase to match the `named` value?